nautobot/extras/views.py

```diff
@@ -1307,7 +1307,8 @@ def get(self, request, class_path=None, pk=None):
     def post(self, request, class_path=None, pk=None):
         job_model = self._get_job_model_or_404(class_path, pk)
 
-        job_form = job_model.job_class.as_form(request.POST, request.FILES) if job_model.job_class is not None else None
+        job_class = get_job(job_model.class_path, reload=True)
```

Comment:
Might seem unnecessary as we *usually* do a GET before a POST to this endpoint, but in the job-button case we don't ever hit the GET side, just POST.